package.json

```diff
@@ -83,8 +84,7 @@
     "node": "12.* || 14.* || >= 16"
   },
   "volta": {
-    "node": "16.20.2",
-    "npm": "6.14.18"
+    "node": "20.18.0"
```

Comment:
I guess, this needs to be in separate PR.

@jelhan mind if we keep `volta` for now as is? I intentionally kept it without changes until v2 conversion is completed

Reply:
The issue was that the existing lock file was not compatible with the specified NPM version. It was in v2 format not supported by NPM v6. Pinning to a recent version and regenerating lock file was the easiest solution.

Reply:
CI was failing. Likely an incompatibility caused by very outdated node 12 used in CI. I restored the original pinned versions of Node and NPM and regenerated the lockfile with that version.